src/components/navigation/navigation.stories.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Navigation from './navigation.svelte'
+  import NavigationItem from './navigationItem.svelte'
+  import NavigationHeader from './navigationHeader.svelte'
+  import NavigationActions from './navigationActions.svelte'
+  import NavigationSeparator from './navigationSeparator.svelte'
+  import NavigationMenu from './navigationMenu.svelte'
+  import Icon from '../icon/icon.svelte'
+  import SlotInfo from '../../storyHelpers/SlotInfo.svelte'
+  import Slot from '../../storyHelpers/Slot.svelte'
+  import Toggle from '../toggle/toggle.svelte'
+</script>
+
+<Meta
+  title="Components/Navigation"
+  component={Navigation}
+  argTypes={{
+    kind: {
```

Comment:
We should probably list all the args here (including CSS props)

Reply:
The navigation component itself doesn't have any CSS props. Some of the sub components have more props/CSS props, but I'm not sure that we'd want to add them here?

Reply:
Hmmm, that's a good point. It'd be good to have some way to figure out all the different knobs for the component, but I guess we can do that separately.